osu.Game/Overlays/Wiki/WikiHeader.cs

```diff
@@ -5,14 +5,16 @@
 using System.Linq;
 using osu.Framework.Bindables;
 using osu.Framework.Graphics;
+using osu.Framework.Localisation;
 using osu.Game.Online.API.Requests.Responses;
+using osu.Game.Resources.Localisation.Web;
 
 namespace osu.Game.Overlays.Wiki
 {
     public class WikiHeader : BreadcrumbControlOverlayHeader
     {
-        private const string index_page_string = "index";
         private const string index_path = "Main_Page";
+        public LocalisableString IndexPageString => LayoutStrings.HeaderHelpIndex;
```

Comment:
static + newline between field and property.
```suggestion

        public static LocalisableString IndexPageString => LayoutStrings.HeaderHelpIndex;
```